package.json

```diff
@@ -37,6 +37,7 @@
     "react": "^16.13.1",
```

Comment:
We should remove the generated files from the PR with `rm -rf docs`. They should still be in version control beceause they'll be added in `gh-pages` branch. Also here's an example of what the npm script should look like:

https://github.com/amilajack/joker/blob/master/package.json#L19

Afte adding this, we can build the docs like so: `yarn docs`

Reply:
Also while we're at it we might as well add a `deploy` npm script similar to https://github.com/amilajack/freesound-player/blob/master/package.json#L47. But rather than building we'd want to build the docs.

Reply:
[`"docs": "rm -rf docs && typedoc ./src --out docs",`](https://github.com/amilajack/joker/blob/110246c98e90de6565400547847bb2235951faf6/package.json#L19)

Small change, but we can remove ` docs` after `typedoc ./src --out` from `"docs": "rm -rf docs && typedoc ./src --out docs",` because the default output directory of TypeDoc is `docs` anyway, right?

Reply:
> Also while we're at it we might as well add a `deploy` npm script similar to https://github.com/amilajack/freesound-player/blob/master/package.json#L47. But rather than building we'd want to build the docs.

`"deploy": "yarn docs && gh-pages -d build",` is what I have for this; is it correct?

Reply:
Actually my bad for backtracking but now that I think about it, if we deploy both storybook and typedoc, the one published second to gh pages will override what was published previously

Reply:
Let's take out deploy script for now.